Fix tests.foreman.cli.test_errata.ErrataTestCase.test_positive_user_permission

https://github.com/SatelliteQE/robottelo/blob/e4eb82e540246c936b3309ca9d669e921f1b3315/tests/foreman/cli/test_errata.py#L1781

On assigning the org to the user the errata was listed, So make the test to respect the org association.
For more details, https://bugzilla.redhat.com/show_bug.cgi?id=1507523#c10

```
>           raise CLIReturnCodeError(*error_data)
E           CLIReturnCodeError: CLIReturnCodeError(return_code=128, stderr=u"[ERROR 2017-11-04 22:08:20 API] 404 Resource Not Found\n[ERROR 2017-11-04 22:08:20 Exception] <NilClass> nil\n\n[ERROR 2017-11-04 22:08:20 Exception] \n\nRestClient::ResourceNotFound (404 Resource Not Found):\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:352:in `block in rest_client_call_block'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:493:in `process_result'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'\n    /opt/rh/rh-ruby23/root/usr/share/ruby/net/http.rb:853:in `start'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:324:in `call_client'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:240:in `http_call'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:190:in `call_action'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:185:in `call'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/resource.rb:21:in `call'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/apipie/command.rb:43:in `send_request'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:166:in `send_request'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:256:in `send_request'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:279:in `execute'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/bin/hammer:147:in `<top (required)>'\n    /usr/bin/hammer:23:in `load'\n    /usr/bin/hammer:23:in `<main>'\n\n\n", msg=u'Command "erratum list" finished with return_code 128\nstderr contains following message:\n[ERROR 2017-11-04 22:08:20 API] 404 Resource Not Found\n[ERROR 2017-11-04 22:08:20 Exception] <NilClass> nil\n\n[ERROR 2017-11-04 22:08:20 Exception] \n\nRestClient::ResourceNotFound (404 Resource Not Found):\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:352:in `block in rest_client_call_block\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:493:in `process_result\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit\'\n    /opt/rh/rh-ruby23/root/usr/share/ruby/net/http.rb:853:in `start\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:324:in `call_client\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:240:in `http_call\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:190:in `call_action\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/api.rb:185:in `call\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/apipie-bindings-0.2.0/lib/apipie_bindings/resource.rb:21:in `call\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/apipie/command.rb:43:in `send_request\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:166:in `send_request\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:256:in `send_request\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli_foreman-0.11.0.3/lib/hammer_cli_foreman/commands.rb:279:in `execute\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/lib/hammer_cli/abstract.rb:29:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run\'\n    /opt/theforeman/tfm/root/usr/share/gems/gems/hammer_cli-0.11.0.1/bin/hammer:147:in `<top (required)>\'\n    /usr/bin/hammer:23:in `load\'\n    /usr/bin/hammer:23:in `<main>\'\n\n\n'
```

--- a/tests/foreman/cli/test_errata.py
+++ b/tests/foreman/cli/test_errata.py
@@ -1713,6 +1713,8 @@ class ErrataTestCase(CLITestCase):
 
         :expectedresults: Check that the new user is able to see errata for one
             product only.
+
+        :BZ: 1403947, 1507523
         """
         user_password = gen_string('alphanumeric')
         user_name = gen_string('alphanumeric')
@@ -1741,7 +1743,9 @@ class ErrataTestCase(CLITestCase):
         user = make_user({
             'admin': False,
             'login': user_name,
-            'password': user_password
+            'password': user_password,
+            'organization-ids': [org['id']],
+            'default-organization-id': org['id'],
         })
         User.add_role({'id': user['id'], 'role-id': role['id']})
         # make sure the user is not admin and has only the permissions assigned

--- a/tests/foreman/cli/test_errata.py
+++ b/tests/foreman/cli/test_errata.py
@@ -1714,7 +1714,7 @@ class ErrataTestCase(CLITestCase):
         :expectedresults: Check that the new user is able to see errata for one
             product only.
 
-        :BZ: 1403947, 1507523
+        :BZ: 1403947
         """
         user_password = gen_string('alphanumeric')
         user_name = gen_string('alphanumeric')

